go-controller/pkg/ovn/obj_retry_master.go

```diff
@@ -0,0 +1,961 @@
+package ovn
+
+import (
+	"fmt"
+	"net"
+	"reflect"
+
+	ocpcloudnetworkapi "github.com/openshift/api/cloudnetwork/v1"
```

Comment:
u need to move this import with other openshift pkgs below

Reply:
I see that `gopls` failed to reorder the imports here... go figure why. :)